src/mock/subscription.ts

```
@@ -0,0 +1,125 @@
+    return {
+      ...subscription,
+      product,
+      plan
+    };
+  }).filter(item => item.product && item.plan);
+};
+
+// 구독하지 않은 상품들을 반환하는 헬퍼 함수
+export const getUnsubscribedProducts = () => {
+  const subscribedProductIds = MockUserSubscriptions
+    .filter(sub => sub.status === "active")
```

ESLint

[error] Replace `⏎····.filter(sub·=>·sub.status·===·"active")` with `.filter(⏎····(sub)·=>·sub.status·===·"active",`

(prettier/prettier)